Make sure screensave.o builds with -fPIC so it can be in screensave.so.

--- elmerfem-5.5.0.svn.4214.dfsg/post/src/Makefile.am~	2008-11-18 08:19:10.000000000 -0500
+++ elmerfem-5.5.0.svn.4214.dfsg/post/src/Makefile.am	2009-06-25 21:05:17.000000000 -0400
@@ -33,6 +33,9 @@
 
 EXTRA_DIST=screensave.c
 
+screensave.$(OBJEXT): screensave.c
+	$(COMPILE) -c $< -fPIC
+
 screensave.so: screensave.$(OBJEXT)
 	$(CC) $(CFLAGS) $(LDFLAGS) $(SH_LDFLAGS) -o $@ $< $(LIBS)
 
